### PR TITLE
HTCONDOR-1708: Documentation for file transfer plugin testing

### PR DESCRIPTION
**Pull request by joereuss12**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [x] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/users-manual/file-transfer.rst [hunk 1]
```diff
@@ -883,3 +883,32 @@ key files described above.
...
+**Testing mode for file transfer plugins**
+
+You can use this modification to test HTCondor file transfer plugins prior to
+being inserted into the list of available URL schemas. If you wish to provide 
+some test links for file transfer plugins, you can add some modifications to 
+the condor_config file. All you need is to add:
+
+.. code-block:: condor-submit
+
+      <URL_TYPE>_TEST_URL = <url_type>://<url>
```
Comment (joereuss12) [inline docs/users-manual/file-transfer.rst:896]: replace <URL_TYPE> with <protocol>

File: docs/users-manual/file-transfer.rst [hunk 2]
```diff
@@ -883,3 +883,32 @@ key files described above.
...
+
+.. code-block:: condor-submit
+
+      HTTPS_TEST_URL = https://google.com
+
+In the above config, HTTPS will only be available if google.com is present. To
+see the list of available url types, just run condor_starter -classad.
+The output of this command should show the line below if all your test url's worked properly:
+
+.. code-block:: condor_submit
```
Comment (joereuss12) [inline docs/users-manual/file-transfer.rst:908]: replace condor_submit with condor-classad

File: docs/users-manual/file-transfer.rst [hunk 3]
```diff
@@ -883,3 +883,32 @@ key files described above.
...
+some test links for file transfer plugins, you can add some modifications to 
+the condor_config file. All you need is to add:
+
+.. code-block:: condor-submit
+
+      <URL_TYPE>_TEST_URL = <url_type>://<url>
+
+Here is an example using https://google.com:
+
+.. code-block:: condor-submit
```
Comment (joereuss12) [inline docs/users-manual/file-transfer.rst:900]: condor-config or something similar

File: docs/users-manual/file-transfer.rst [hunk 4]
```diff
@@ -883,3 +883,32 @@ key files described above.
 
 If you need to specify a region, you may do so using ``aws_region``,
 despite the name.
+
+**Testing mode for file transfer plugins**
+
+You can use this modification to test HTCondor file transfer plugins prior to
```
Comment (joereuss12) [inline docs/users-manual/file-transfer.rst:889]: This option allows us to test against a file transfer protocol. This is used to determine if the condor starter has the capabilities to use this protocol. Any failed tests will be removed from the list of "HasFileTransferPlugins", any successful will make no change to that list.

File: docs/users-manual/file-transfer.rst (hunk 1 above)
Comment (joereuss12) [reply docs/users-manual/file-transfer.rst:896]: remove above example

**Comment by ColeBollig** [comment]
Closing to re-open a better branch to merge (issues with commits and such

**Review by ColeBollig — APPROVED** [review]
LGTM

**Pull request merged**